source/_integrations/samsungtv.markdown

```diff
@@ -51,9 +51,11 @@ turn_on_action:
 
 After saving the YAML configuration, the TV must be turned on _before_ launching Home Assistant in order for the TV to be registered the first time.
 
-#### Wake up TV
+#### Wake up TV / TV does not turn on
 
-To wake up the TV when switched off you can use the [wake-on-lan](/integrations/wake_on_lan/) integration and call a service. This is not possible with every device.
+If the integration knows the MAC address of the TV from discovery, it will attempt to wake it using wake on lan when calling turn on. Wake on lan must be enabled on the TV for this to work. If the TV is connected to a smart strip or requires a more complex turn on process, a `turn_on_action` can be provided that will take precedence over the built-in wake on lan functionality.
```

Comment:
```suggestion
If the integration knows the MAC address of the TV from discovery, it will attempt to wake it using wake on LAN when calling turn on. Wake on LAN must be enabled on the TV for this to work. If the TV is connected to a smart strip or requires a more complex turn-on process, a `turn_on_action` can be provided that will take precedence over the built-in wake on LAN functionality.
```